src/editor/Model/Reducer.re

```diff
@@ -29,20 +29,32 @@ let showTablineTabs = (state: State.t, tabs) =>
   };
 
 let showTablineBuffers = (state: State.t, buffers, activeBufferId) =>
-  switch (state.configuration.editorTablineMode) {
-  | Buffers =>
+  switch (state.configuration.editorTablineMode, activeBufferId) {
+  | (Buffers, Some(activeId)) =>
     List.map(
       ({id, filePath, modified, _}: BufferMetadata.t) =>
         State.Tab.{
           id,
           title: filePath |> truncateFilepath,
-          active: activeBufferId == id,
+          active: activeId == id,
+          modified,
+        },
+      buffers,
+    )
+  | (Buffers, None) =>
+    List.map(
+      ({id, filePath, modified, _}: BufferMetadata.t) =>
+        State.Tab.{
+          id,
+          title: filePath |> truncateFilepath,
+          active: false,
           modified,
         },
       buffers,
     )
     |> sortTabsById
-  | _ => state.tabs
+  | (Tabs, _) => state.tabs
+  | (Hybrid, _) => state.tabs
```

Comment:
What does `Hybrid` mean?

Reply:
@bryphe this predates this PR I just took the opportunity to clear with the exhaustiveness check